package.json

```diff
@@ -11,11 +11,16 @@
   "bin": {
     "pagespeed-score": "cli.js"
   },
+  "xo": {
+    "space": true,
+    "env":["node"]
```

Comment:
Default for xo as per documentation => [Documentation](https://github.com/sindresorhus/xo#space) is
``` js
"space" : false
```

Yeah env default is node and it can be ommiteed.